sam-app/lambda_functions/template.yaml

```diff
@@ -1375,7 +1379,7 @@ Resources:
       FunctionName: 
         Fn::GetAtt: sfCTRTrigger.Arn
       StartingPosition: "LATEST"
-      BatchSize: 100
+      BatchSize: !Ref CTREventSourceMappingBatchSize
```

Comment:
nice!

Reply:
make sure we add this param in documentation.